content/resources.json

```diff
@@ -98,7 +98,7 @@
       ]
     },
     {
-      "title": "Confidential Computing",
+      "title": "CC",
```

Comment:
issue, suggestion(blocking): I think CC is confusing to newcomers. I suggest sticking with Confidential Computing as this is precise and clear for everyone. Let me know if this works for you or if I am missing something.